src/ucp/api/ucp.h

```diff
@@ -251,9 +251,28 @@ enum ucp_ep_close_mode {
                                               for all endpoints created on
                                               both (local and remote) sides to
                                               avoid undefined behavior. */
-    UCP_EP_CLOSE_MODE_FLUSH         = 1  /**< @ref ucp_ep_close_nb schedules
+    UCP_EP_CLOSE_MODE_FLUSH         = 1, /**< @ref ucp_ep_close_nb schedules
                                               flushes on all outstanding
                                               operations. */
+    UCP_EP_CLOSE_MODE_STREAM_SYNC   = 2  /**< @ref ucp_ep_close_nb will flush
```

Comment:
IMHO STREAM_SYNC is not a good name , it does not indicate that the stream is closed/disconnected
maybe STREAM_END / STREAM_FIN ?